pages/common/cmake.md

```diff
@@ -1,13 +1,21 @@
 # cmake
 
-> Cross-platform build system generator.
-> It generates Makefiles, Visual Studio projects or others, depending on the target system.
+> Cross-platform build, testing and packaging automation system.
+> CMake uses own shell-like syntax and generates recipes for several build systems.
```

Comment:
```suggestion
> CMake uses its own shell-like syntax and generates recipes for many build systems.
```

Reply:
@bl-ue, I am not sure cause I'm not native, but I hear "-s" possessive form is unapplicable to inanimate nouns, or does it?

Reply:
I can't remember :smile:, but I am native and I know that "its" in this sentence is correct.

Reply:
Well, I will remember this, thanks!